.github/ISSUE_TEMPLATE.md

```diff
@@ -26,7 +26,7 @@
 
 ## Your Environment
 <!--- Include relevant details about the environment you experienced the bug in -->
-* KeePassXR version/commit used: 
+* KeePassXC version/commit used: 
```

Comment:
I would point out here with a comment how to get the version: About Screen -> revision